scripts/bootstrap.sh

```diff
@@ -5,12 +5,13 @@ set -exo pipefail
 source ./scripts/precheck.sh
 
 
-go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.25
-go install github.com/mikefarah/yq/v4@v4.27.5
+go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0
```

Comment:
> Is v1.25 available in Ubuntu?

Yes I think it's all available. It's platform irrelevant, no?

> if the developers are using different versions of the toolchain, the generated protobuf code will keep changing.

It will be better if we use asdf or a similar tool. With asdf and corresponding plugins, we can simple specify the versions in `<project-root>/.tool-versions`:

```
golang 1.21.1
protoc 24.3
yq 4.35.2
grpcurl 1.8.8
protoc-gen-go 1.31.0
protoc-gen-go-grpc 1.3.0
```

That's how I manage tools versions, asdf works for most shell environments on Mac, Linux, and WSL.

Reply:
But then the asdf will be a requirements; if someone doesn't want to use it, he/she will have to install corresponding tools system-wide with the versions specified in this file.